### PR TITLE
Fix to enable multi-store order cancellation

### PR DESCRIPTION
**Pull request by vitormattos**
# Problem:
If the magento installation has two or more stores enabled, only is possible cancel orders from store 01.

# Technical solution:
The cancellation flow was taking the merchantKey from store 01
Remove the constructor from children classes and corretly inheritance implement

### CODE REVIEW

**Comment by felipebool** [comment]
We are following a PR template which is something like:

What?
Describe the problem

Why?
Describe the motivation

How?
Describe the process

Please, follow this template in the next PR

File: app/code/community/Uecommerce/Mundipagg/Model/Api.php [hunk 1]
```diff
@@ -42,9 +42,9 @@ class Uecommerce_Mundipagg_Model_Api extends Uecommerce_Mundipagg_Model_Standard
 	private $debugEnabled;
 	private $moduleVersion;
 
-	public function __construct() {
+	public function __construct($Store) {
```
Comment (felipebool) [inline app/code/community/Uecommerce/Mundipagg/Model/Api.php]: PSR-2

"Opening braces for classes MUST go on the next line, and closing braces MUST go
on the next line after the body"

Please, review this

File: app/code/community/Uecommerce/Mundipagg/Model/Standard.php [hunk 2]
```diff
@@ -173,38 +173,33 @@ public function getCieloSku() {
 		return $this->cieloSku;
 	}
 
-	public function __construct() {
-		$this->setEnvironment($this->getConfigData('environment'));
-		$this->setPaymentAction($this->getConfigData('payment_action'));
-
-		switch ($this->getConfigData('environment')) {
+	public function __construct($Store = null) {
```
Comment (felipebool) [inline app/code/community/Uecommerce/Mundipagg/Model/Standard.php]: PSR-2

"Opening braces for methods MUST go on the next line, and closing braces MUST
go on the next line after the body."

Please, review this

File: app/code/community/Uecommerce/Mundipagg/Model/Standard.php [hunk 3]
```diff
@@ -173,38 +173,33 @@ public function getCieloSku() {
...
-				$this->setDebug($this->getConfigData('debug'));
-				$this->setEnvironment($this->getConfigData('environment'));
-				$this->setCieloSku($this->getConfigData('cielo_sku'));
+			    $environment = 'Production';
 				break;
-		}
+	    }
+	    $this->setmerchantKey(trim($this->getConfigData('merchantKey'.$environment, $Store)));
+	    $this->setUrl(trim($this->getConfigData('apiUrl'.$environment, $Store)));
+		$this->setPaymentAction($this->getConfigData('payment_action', $Store));
```
Comment (felipebool) [inline app/code/community/Uecommerce/Mundipagg/Model/Standard.php]: Remove extra space before $this

File: app/code/community/Uecommerce/Mundipagg/Model/Standard.php (hunk 2 above)
Comment (vitormattos) [inline app/code/community/Uecommerce/Mundipagg/Model/Standard.php]: PSR-2 will applied in another pull request

**Pull request merged**